test/IntrusivePtr.cpp

```diff
@@ -0,0 +1,198 @@
+#include "pch.h"
+#include <kf/stl/new>
+#include <kf/boost/intrusive_ptr.hpp>
+#include <kf/boost/intrusive_ref_counter.hpp>
+#include <kf/UString.h>
+
+struct IntrusivePtrTestStruct : public boost::intrusive_ref_counter<IntrusivePtrTestStruct>
+{
+    kf::UString<PagedPool> str;
```

Comment:
Why do we need 2 members in the struct? It seems to be just extra noise with no real value.

Reply:
removed string